prereise/gather/griddata/hifld/data_process/tests/test_transmission.py

```diff
@@ -185,71 +180,49 @@ def test_create_transformers():
         },
```

Comment:
Not in this PR, it seems bus with index 7 is not used in this test case?

Reply:
Correct. It's there to make sure that the code can handle substations with only one bus in them.